vpr/src/route/route_timing.cpp

```diff
@@ -1211,12 +1211,14 @@ static bool timing_driven_route_sink(ClusterNetId net_id,
     bool net_is_global = cluster_ctx.clb_nlist.net_is_global(net_id);
     bool high_fanout = is_high_fanout(cluster_ctx.clb_nlist.net_sinks(net_id).size(), high_fanout_threshold);
     constexpr float HIGH_FANOUT_CRITICALITY_THRESHOLD = 0.9;
+    constexpr size_t HIGH_FANOUT_CONNECTIONS_ROUTED_THRESHOLD = 5;
     bool sink_critical = (cost_params.criticality > HIGH_FANOUT_CRITICALITY_THRESHOLD);
+    bool many_connections_routed = router_stats.connections_routed > HIGH_FANOUT_CONNECTIONS_ROUTED_THRESHOLD;
```

Comment:
What are the units on `router_stats.connections_routed`?  I'm wondering whether you should use iteration number fraction, like the high effort routing does?  E.g. only all high fanout on the first 25% of iterations, or something.

Reply:
It's just a count. I've only seen it get stuck when the connection & nets re-routed are in the low single digits. I could use the delta, but if it's worth putting a lot of effort in, I'd prefer to figure out why it gets stuck instead.

Reply:
Wait a second, is this logic backwards then?  This logic will disable high fanout routing when the number of connections routed is low, and enabled it once more than 5 have been routed.  Is that the intention?

Reply:
Yeah, the idea is that high fanout routing is an optimization, so not running it for the first few connections wouldn't be noticable.

I'm working on a new method that disables high fanout routing when the delta hits zero, which should avoid the need for yet another parameter.